infrastructure/firecracker/config.go

```diff
@@ -88,7 +89,11 @@ func WithMicroVM(vm *models.MicroVM) ConfigOption {
 			})
 		}
 
-		kernelArgs := vm.Spec.Kernel.CmdLine
+		kernelArgs := defaults.KernelCmdLine
+
+		if vm.Spec.Kernel.CmdLine != "" {
```

Comment:
This logic can then be generic (i.e. live outside of the firecracker provider) to flintlock and use the defaults from the provider.